baseplate/lib/edge_context.py

```diff
@@ -19,15 +28,14 @@ class NoAuthenticationError(Exception):
 class AuthenticationTokenValidator:
     """Factory that knows how to validate raw authentication tokens."""
 
-    def __init__(self, secrets):
+    def __init__(self, secrets: SecretsStore):
         self.secrets = secrets
 
-    def validate(self, token):
+    def validate(self, token: bytes) -> "AuthenticationToken":
```

Comment:
Not sure how much we care about this but would it make sense to just move the class definition for this up so we don't have to use a string?

Reply:
I think it's probably fine to leave like this. In 3.8 (or 3.7 w/ `from __future__ import annotations`) _all_ annotations become strings without having to quote them so ordering doesn't matter.